README.md

```diff
@@ -303,6 +303,16 @@ To enable inner join add this to your configuration.
 ```
 
 
+There is also an ORM Query Type for LeftJoin.  This join type is commonly used to fetch an empty right side of a relationship.
+
+Left Join is not included by default in the `zf-doctrine-querybuilder.global.php.dist`.
```

Comment:
Fine. Don't you think then we need add here how to enable it? I know it will be very similar to `innerjoin`, but maybe better to have it here, and if someone want enable it can just copy-paste this part from the docs. Or maybe even better, add it to .dist config but commented out?